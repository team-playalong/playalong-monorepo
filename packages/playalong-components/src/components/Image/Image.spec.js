import * as React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import PlyImage from './Image';

let wrapper;
const props = {
  src: 'www.image.com',
};

describe('Spinner Component', () => {
  beforeEach(() => {
    wrapper = shallow(<PlyImage {...props} />);
  });

  test.skip('Should have a source', () => {
    expect(wrapper.props().src).toBe(props.src);
  });

  test.skip('Snapshot testing', () => {
    const tree = renderer.create(<PlyImage {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
