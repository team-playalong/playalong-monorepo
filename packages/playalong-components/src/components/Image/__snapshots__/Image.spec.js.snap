// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component Snapshot testing 1`] = `
<img
  alt="No Image Available"
  src="www.image.com"
/>
`;
